<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextBrowser" name="logBrowser">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>11</y>
        <width>781</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Records_Check">
     <attribute name="title">
      <string>Records Check</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Check Records</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="domainLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Domain: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Contains spfva.gofax.com.au: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Contains DKIM Key 1: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Contains DKIM Key 2: </string>
      </property>
     </widget>
     <widget class="QLabel" name="domainLabelResult">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>611</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>domain</string>
      </property>
     </widget>
     <widget class="QLabel" name="spfLabelResult">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>611</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SPF</string>
      </property>
     </widget>
     <widget class="QLabel" name="DKIMLabelResult_1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>611</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DKIM Key 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="DKIMLabelResult_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>611</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DKIM Key 2</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionImport_Modules"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionImport_Modules">
   <property name="text">
    <string>Import Modules</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
